source/slang/hlsl.meta.slang

```diff
@@ -2504,131 +2649,131 @@ __generic<T : __BuiltinIntegerType>
 __glsl_extension(GL_KHR_shader_subgroup_arithmetic)
 __spirv_version(1.3)
 __target_intrinsic(glsl, "subgroupAnd($0)")
-__target_intrinsic(cuda, "_waveAnd($0)")
+__target_intrinsic(cuda, "_waveAnd(_getActiveMask(), $0)")
 T WaveActiveBitAnd(T expr);
```

Comment:
It doesn't have to be in this PR, but I think it would be more correct to have the existing `WaveActive*()` functions be defined to have `__target_intrinsic` mappings for HLSL/GLSL, and then to have an explicit body as a fallback:

```hlsl
T WaveActiveBitAnd<T>(T expr)
{
    return WaveMaskBitAnd(WaveGetActiveMask(), expr);
}
```

Here I assume that `WaveGetActiveMask()` is a version that corresponds to the HLSL "active" notion (so `WaveActiveBallot(true)`). This is the the Right Way to have this operations be defined for CUDA, since it says that a `WaveActive*()` function should map to a call to the corresponding `WaveMask*()` function using the HLSL "active" mask.

The benefit of such a stdlib definition will come down the line. For now it would be equivalent to what you have here, since our near-term CUDA version of `WaveGetActiveMask()` will probably need to use `__activeMask()`.

In the longer run, once we implement a correct IR translation to synthesize the value of the HLSL "active" mask for CUDA and other similar targets, these `WaveActive*()` functions would Just Work.